torchgen/gen.py

```diff
@@ -1829,6 +1838,17 @@ def gen_aten_interned_strings() -> Dict[str, str]:
 
     core_fm.write("aten_interned_strings.h", gen_aten_interned_strings)
 
+    def gen_tags_enum() -> Dict[str, str]:
+        return {
+            "enum_of_valid_tags": (
+                "enum Tags\n{\n"
```

Comment:
Use an enum class; with enum they get dumped in global namespace

Reply:
you might also want to dump the actual enum declaration bit directly inside of the template. That way someone staring at the template has a better idea of what the file is for without having to jump to codegen